src/wmtk/invariants/MultiMeshTopologyInvariant.hpp

```diff
@@ -0,0 +1,23 @@
+#include "MeshInvariant.hpp"
+
+namespace wmtk {
+class MultiMeshEdgeTopologyInvariant : public MeshInvariant
+{
+public:
+    MultiMeshEdgeTopologyInvariant(const Mesh& parent, const Mesh& child, const PrimitiveType pt);
```

Comment:
if the child mesh must be an EdgeMesh why not check for that explicitly instead of passing a generic `Mesh` ref?